front/app/modules/commercial/project_folders/citizen/components/ProjectFolderGoBackButton/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Excellent work on the test! 🚀 
Conventionally, we name the test file with the name of the component we are testing: `ProjectForlderGoBackButton.test.tsx`